src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -301,7 +301,7 @@
     <value>Setting the ViewStateUserKey property can help you prevent attacks on your application by allowing you to assign an identifier to the view-state variable for individual users so that they cannot use the variable to generate an attack. Otherwise, there will be cross-site request forgery vulnerabilities.</value>
   </data>
   <data name="SetViewStateUserKeyMessage" xml:space="preserve">
-    <value>The subclass {0} of Page does not set the ViewStateUserKey property in method OnInit</value>
+    <value>The subclass {0} of Page does not set the ViewStateUserKey property in either method OnInit or method Page_Init</value>
```

Comment:
>The subclass {0} of Page does not set the ViewStateUserKey property in either method OnInit or method Page_Init [](start = 11, length = 111)

The class {0} derived from System.Web.UI.Page does not set the ViewStateUserKey property in the OnInit method or Page_Init method.